content/influxdb/cloud/account-management/switch-org.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Switch InfluxDB Cloud organizations
+seotitle: Switch between InfluxDB Cloud organizations
+description: >
+  Switch from one InfluxDB Cloud account to another and set a default account.  
+menu:
+  influxdb_cloud:
+    name: Switch InfluxDB accounts
+    parent: Account management
+weight: 105
+---
+If you belong to more than one {{< cloud-name >}} organization with the same email address, you can switch from one organization to another while staying logged in. An account can contain multiple organizations. 
```

Comment:
Let's add an intro on orgs:

> Organizations are isolated environments for time series data and the resources to make use of it, e.g. tasks, dashboards, etc. InfluxDB Cloud accounts can have one or more organizations. Organizations can only belong to a single InfluxDB Cloud account.

> If your InfluxDB Cloud user belongs to more than one {{< cloud-name >}} organization, you can switch from one organization to another using the following steps:

Reply:
Used the wording from our existing org docs instead.